dap-mode.el

```diff
@@ -137,8 +137,31 @@ The hook will be called with the session file and the new set of breakpoint loca
 (defcustom dap-debug-template-configurations nil
   "Plist Template configurations for DEBUG/RUN."
   :safe #'listp
+  :group 'dap-mode
   :type '(plist))
 
+(defcustom dap-auto-configure-features '(sessions locals breakpoints expressions controls tooltip)
+  "Windows to auto show on debugging when in dap-ui-auto-configure-mode."
+  :group 'dap-mode
+  :type '(set (const sessions)
+              (const locals)
+              (const breakpoints)
+              (const expressions)
+              (const controls)
+              (const tooltip)))
+
+(defconst dap-feature->start-stop
```

Comment:
nit: what's the meaning of `->` here?

Reply:
It's just that given a feature, it returns the start-stop functions? 
I'm open to suggestions for a better name :sweat_smile:

Reply:
Since this is used to showing windows, how about `dap-features->windows`?

Reply:
Looks good, gonna change :)